apps/hpl/2.1/8proc_hpl_example.sh.tpl

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -l
+#@      cw_TEMPLATE[name]="HPL v2.1 Template (GridScheduler)"
+#@      cw_TEMPLATE[desc]="8 process MPI example script."
+#@ cw_TEMPLATE[copyright]="Copyright (C) 2016 Alces Software Ltd."
+#@   cw_TEMPLATE[license]="Creative Commons Attribution-ShareAlike 4.0 International"
+#==============================================================================
+# Copyright (C) 2016 Alces Software Ltd.
+#
+# This work is licensed under a Creative Commons Attribution-ShareAlike
+# 4.0 International License.
+#
+# See http://creativecommons.org/licenses/by-sa/4.0/ for details.
+#==============================================================================
+
+#############################################
+# SGE Directives - Change as required
+#############################################
+
+#$ -j y -N hpl -o $HOME/hpl_out.$JOB_ID
+#$ -cwd -V -pe mpislots-verbose 8
+
+
+#############################################
+# APPLICATION COMMANDS - change as required
+#############################################
+#
+# For more information about configuring HPL see:
+#   http://www.netlib.org/benchmark/hpl/
+
+# Load module
+module load apps/hpl
```

Comment:
This should probably contain an explicit version -- otherwise it may load the wrong `apps/hpl`.